trame_vtk/modules/vtk/serializers/actors.py

```diff
@@ -60,6 +61,16 @@ def generic_actor_serializer(parent, actor, actor_id, context, depth):
                 dependencies.append(texture_instance)
                 calls.append(["addTexture", [wrap_id(texture_id)]])
 
+    # Apply transform
+    if actor.GetUserMatrix():
+        user_matrix = [0] * 16
+        matrix = actor.GetUserMatrix()
+        for i in range(4):
+            for j in range(4):
+                idx = i + 4 * j
```

Comment:
Not sure but we may need to transpose the matrix due to different storage in vtk.js vs VTK.

Reply:
normally the for loop is doing it, vtk.js uses column major raveled matrices if I understood it well, so `matrix.GetElement(i,j)` is element `i + 4*j`. The mesh transformation show the serialization is good.